crux-test/test/crux/api/JCruxNodeTest.java

```diff
@@ -107,66 +96,66 @@ public void statusTest() {
 
     @Test(expected = NodeOutOfSyncException.class)
     public void hasTxCommittedThrowsTest() {
-        TransactionInstant tx = put();
+        TransactionInstant tx = p();
```

Comment:
not a fan of single-letter methods - could this go back to `put`?

Reply:
Yep. This was a holdover from when it was conflicting with the static import of TestUtils. This test now uses `TestUtils.tx` instead so I've changed it to the full word.